.gitignore

```diff
@@ -14,3 +14,10 @@
 __pycache__/
 *.pyc
 *.pyo
+
+venv
+
+# Secrets
```

Comment:
@quick007 perhaps these would be better off in a "docker/secrets" folder and we could ignore the whole path in case we add more?